### DISCUSSION TITLE
Is Preview reflecting actual settings?

### DISCUSSION THREAD

**Discussion by mpek** [RandomEngy/VidCoder#1065] [Ideas]
I recognized a color shift (lighter in rendered preview, but not in final output, euh?), is it possible to offer a live preview of settings to find the right settings for filters quicker? 
I'm with Vidcoder 7.15.

**Comment by RandomEngy** [comment]
Yeah the static previews can be generated very quickly but  don't reflect all encoding settings. They are most useful for picture size adjustments.

You pretty much need to do an actual encode to see what it will really look like. You can encode a preview clip to do that.

Any change in the static previews would need to be supported by the HandBrake API, which VidCoder uses.

**Reply by mpek** [reply]
Thank you for your fast reply and clarification. Wouldn't it be nice to be able to see changed settings immediately? Wonder why it is not implemented yet.

**Reply by RandomEngy** [reply]
Encoding a video is a complex process and to save data each frame only shows differences from the last encoded frame. Which means if you want to accurately show what a frame looks like, you actually need to encode a little chunk of video. It would be slow and you'd have a lot of throwaway work. You'd also have to try and guess which frame is the one you want from the little video chunk; there are different frame timings and it doesn't always line up cleanly for videos with variable frame rate.
